src/components/Practice/Mechanics7.jsx

```
@@ -0,0 +1,772 @@
+                  borderRadius: "12px",
+                  padding: answer === "correct" ? "0px" : "5px 10px 5px 10px",
+                  border: getBorder(),
+                  color: colors,
+                  fontWeight: type === "word" ? 600 : 700,
+                  fontSize: "35px",
+                  fontFamily: "Quicksand",
+                  cursor: "pointer",
+                  marginLeft:
+                    type === "word" ? 0 : answer != "correct" ? "20px" : 0,
+                }}
+              >
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)